### PR TITLE
XML CS fixes

### PR DESCRIPTION
**Pull request by wojsmol**
Pull Request for Issue joomla#12080
### Summary of Changes

XML CS for joomla#12080

### CODE REVIEW

File: administrator/components/com_config/model/form/application.xml [hunk 1]
```diff
@@ -6,33 +6,33 @@
...
 			name="caching"
 			type="list"
-			default="2"
 			label="COM_CONFIG_FIELD_CACHE_LABEL"
 			description="COM_CONFIG_FIELD_CACHE_DESC"
+			default="2"
 			filter="integer"
 			>
 			<option value="0">COM_CONFIG_FIELD_VALUE_CACHE_OFF</option>
-			<option value="1">COM_CONFIG_FIELD_VALUE_CACHE_CONSERVATIVE</option>
```
Comment (zero-24) [inline administrator/components/com_config/model/form/application.xml:15]: Why did you change the ordering here? It looks good to me bevor the change ;) but maybe i miss something :)

Comment (wojsmol) [reply administrator/components/com_config/model/form/application.xml:15]: fixed in https://github.com/zero-24/joomla-cms/pull/17/commits/f74f7ebf6ea29666808bd006c87038fdb722004f

**Comment by zero-24** [comment]
Thanks 👍
